Fix inconsistencies in UG/DG (#139)
AY1920S1-CS2103T-T13-4/main: docs/UserGuide.adoc

.Mark's Offline Tab View
image::ui-screenshots/uimockup1_offline.png[@Offline Tab View, 600, 400]

_{ more coming soon }_

> == Glossary

Reviewer: Maybe we can reorder the glossary? Like _Bookmark_ -> _Offline-copy_ -> _Annotation_ -> _Folder Hierarchy_ -> _Dashboard_. Since some of the definition depends on other definition.

Author: I did consider that, but I think it's okay because the glossary isn't meant to be read in order. It's currently arranged in alphabetical order so that the user can easily find a word/phrase that they're looking for.

Reviewer: Okay

This glossary aims to provide a definition for the special vocabulary used in this user guide.

[[annotation]] Annotation::

> Notes or highlights that can be made on <<offline-copy, offline copies>> of a bookmark.

Reviewer: ```suggestion
Notes or highlights of <<offline-copy, offline copies>>.
```
I think don't need to add "can be" because it kind of increase the uncertainty of the definition. (Compare to other definition)

Author: Ok!
```suggestion
A note or highlight that is made on an <<offline-copy, offline copy>> of a bookmark.
```

[[bookmark]] Bookmark::
A saved shortcut to a website. In Mark, a bookmark can contain other information besides the website URL,
such as reminders, <<offline-copy, offline copies>> and <<annotation, annotations>>.

[[dashboard]] Dashboard::
The default tab in view when the application starts. The dashboard
contains a help section (command summary), favorite tags, current
reminders, and a view of the folder hierarchy for bookmarks.

[[folder-hierarchy]] Folder Hierarchy::
The structure of folders in Mark. This is displayed in the dashboard tab when the application starts.

[[offline-copy]] Offline copy::

> An offline copy of a bookmarked website that is stored as a HTML file on the computer.

Reviewer: ```suggestion
The copy of a bookmarked website that is used for offline viewing. It is stored as a HTML file on the computer.
```
I think can explain about the offline part, so the reader can know better about the purpose of offline-copy.
